src/ol3cesium.js

```diff
@@ -340,7 +346,7 @@ olcs.OLCesium.prototype.updateTrackedEntity_ = function() {
     bs.radius = 1;
   }
   this.entityView_ = new Cesium.EntityView(trackedEntity, scene, scene.mapProjection.ellipsoid);
-  this.entityView_.update(Cesium.JulianDate.now(), bs); // FIXME: have a global management of current time
+  this.entityView_.update(this.time_, bs); // FIXME: have a global management of current time
```

Comment:
Instead of a constant time I would expect a function returning a time.
The application could then choose to:
- pass Cesium.JulianDate.now (current behaviour);
- pass a function returning a constant time (what you propose);
- pass a function driven by a UI component (similar to the time component in Cesium).